extensions/two-factor-auth/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "two-factor-auth",
+  "title": "Two-Factor Auth",
```

Comment:
@kylemh made valid point that this naming might be a bit too generic. Any ideas on having a better extension name here so it's more clear to users in the store what it does? "Two-factor Authentication Code Generator"?